src/helper/horizon-rpc.ts

```diff
@@ -249,11 +249,13 @@ export const submitTransaction = async (
   error: unknown;
 }> => {
   const TxBuilder =
-    networkPassphrase === StellarSdk.Networks.FUTURENET
+    networkPassphrase === StellarSdk.Networks.FUTURENET ||
+    networkPassphrase === StellarSdk.Networks.TESTNET
```

Comment:
it might be useful to create a helper for this since you're doing this check a lot. It'll help guard against places where you accidentally forgot to check both networks.

Reply:
Yeah for sure, I can pull this off into a helper 👍

Reply:
@piyalbasu check out 4547bad969816bf8ba321f1b5aeb99e9864341a8
I added the `getSdk` helper and used it for every sdk instance.